.travis.yml

```diff
@@ -4,7 +4,7 @@ before_install:
   - brew update
   - brew install opam
   - opam init
-  - opam switch 4.02.3
+  - opam switch 4.04.0
```

Comment:
We can set up tests for each version 4.02.3, 4.03, 4.04 for every commit, as a next step. For now, I think we discussed offline that we'd keep everything at 4.02.3 and down-convert from the 4.04 AST that we parse into.